awscli/customizations/s3/subcommands.py

```diff
@@ -369,6 +369,11 @@
                         'output is suppressed.')}
 
 
+NO_PROGRESS = {'name': 'no-progress', 'action': 'store_true',
```

Comment:
The `no-*` params typically have a `store_false` action with the destination being the param without the `no` prefix (`progress` in this case).  Also makes it less confusing if we ever add an explicit true version of this arg (`--progress`).